ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -205,11 +208,39 @@ enum CocoaPodUtils {
         }
       }
     }
+    // Organize the subspecs
+    var versions: [String: String] = [:]
+    var subspecs: [String: Set<String>] = [:]
+
+    for (podName, version) in pods {
+      let subspecArray = podName.components(separatedBy: "/")
+      if subspecArray.count > 2 {
+        fatalError("Multi-layered subspecs are not supported - \(podName)")
+      } else if subspecArray.count == 1 {
+        versions[podName] = version
+      } else {
+        if let previousVersion = versions[podName], version != previousVersion {
+          fatalError("Different installed versions for \(podName)." +
+            "\(version) versus \(previousVersion)")
+        } else {
+          let basePodName = subspecArray[0]
+          versions[basePodName] = version
+          if subspecs[basePodName] == nil {
+            subspecs[basePodName] = Set()
+          }
+          subspecs[basePodName]?.insert(subspecArray[1])
```

Comment:
Here and below - you may be able to clean it up with the `subscript(_:default:)` call:

`subspecs[basePodName, default: Set()].insert(subspecArray[1])`

This will use the `default` value if it's currently `nil`.

Reply:
Much cleaner. Thanks! Done.